queries.sql

```diff
@@ -1,3 +1,58 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT 
+* 
+from 
+animals 
+WHERE 
+name LIKE '%mon';
+
+SELECT
+name
+FROM 
+animals
+ WHERE
+ date_of_birth BETWEEN 2016 AND 2019;
+
+ SELECT
+ name
+ FROM
+ animals
+ WHERE
+ neutered = true AND escape_attempts < 3;
+
+ SELECT
+ date_of_birth
+ FROM
+ animals
+ WHERE
+ name IN ('Agumon','Pikachu');
+
+ SELECT
+ name,
+ escape_attempts
+ FROM
+ animals
+ WHERE
+ weight_kg > 10.5;
+
+ SELECT
+ *
+ FROM
+ animals
+ WHERE
+ neutered = true;
+
+ SELECT
+ *
+ FROM
+ animals
+ WHERE
+ name <> Gabumon;
```

Comment:
- Good job done so far, but according to the project requirements, you are to attach the screenshots for all queries, I cannot find the screenshot for this query. Please fix this 🙏